### ISSUE TITLE
Profile / Autosave: System.IO.FileNotFoundException

### ISSUE THREAD

**Issue by akuropka** [cyanfish/naps2#452]
**Describe the bug**
I am using NAPS2 with autosave enabled in the profile settings.
Sometimes it happens that scans are not saved. This may be a single page or some pages of a batch scan.
A preview is still seen in the GUI but it cannot be viewed.

**To Reproduce**
Scan something and see the progress.

**Expected behavior**
Page is saved as set up.

**Screenshots**
![20240918_153624_explorer](https://github.com/user-attachments/assets/90c2a27a-f8b5-4dfa-a4e6-98f0ec9d9445)

**Desktop**
Microsoft Windows [Version 10.0.26120.1542]

**Additional context**
[errorlog.txt](https://github.com/user-attachments/files/17044964/errorlog.txt)

**Comment by nonubot123** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by cyanfish** [comment]
Are you still experiencing this issue? Can you show me your autosave settings, and what version of NAPS2 are you using?

**Comment by akuropka** [comment]
Yes, the issue still persists, just tested again. I am using the latest 7.5.3.

```
<AutoSaveSettings>
      <FilePath>C:\Temp\$(YYYY)-$(MM)-$(DD)\$(nnnn).jpg</FilePath>
      <PromptForFilePath>false</PromptForFilePath>
      <ClearImagesAfterSaving>false</ClearImagesAfterSaving>
      <Separator>FilePerPage</Separator>
</AutoSaveSettings>
```
Thank you!
